libsovtoken/src/utils/callbacks.rs

```diff
@@ -0,0 +1,133 @@
+//!
+
```

Comment:
is this file even used any more?

Reply:
yes. these functions use in `logic/indy_sdk_api` a lot to call indy-sys functions